opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -17,12 +17,23 @@ syntax = "proto3";
 package opentelemetry.proto.trace.v1;
 
 import "opentelemetry/proto/common/v1/common.proto";
+import "opentelemetry/proto/resource/v1/resource.proto";
 
 option java_multiple_files = true;
 option java_package = "io.opentelemetry.proto.trace.v1";
 option java_outer_classname = "TraceProto";
 option go_package = "github.com/open-telemetry/opentelemetry-proto/gen/go/trace/v1";
 
+// A collection of spans from a Resource.
+message ResourceSpans {
```

Comment:
I'm curious if order matters here? Would it be better for a very long collection of spans to read resource first and them stream thru the collection? It may be a dumb question as I was never tried to optimize anything proto-related

Reply:
Actually even logically the proposed order makes more sense, don't know if a real optimization but makes sense